hazelcast/src/test/java/com/hazelcast/client/config/AbstractClientConfigBuilderTest.java

```diff
@@ -555,4 +555,7 @@ public void testSqlConfigs() {
 
     @Test
     public abstract void testCompactSerialization_withInvalidCompactSerializableClass();
+
+    @Test
+    public abstract void testAlto();
```

Comment:
Can't we just write logic here using `fullClientConfig`? I think for compact tests we didn't because it is impossible to cover all cases with one config, but we can do for alto. I think we can also test the default value with `defaultClientConfig`

One more thing, I noticed `hazelcast-client-default.xml` uses 5.2 version, should we update it to 5.3 like you did for other files?